valora-inc/wallet
# fix(nav): app crash on procrastinate recovery

### Description

On iOS when a user would procrastinate during the recovery phrase setup flow the app would crash.

#### Video Before

https://github.com/valora-inc/wallet/assets/26950305/1f5cd7d6-cfee-41c5-a74a-633e4efd954c

#### Video After

https://github.com/valora-inc/wallet/assets/26950305/387adf69-79d7-4e0c-b10a-2f6591dec421

### Test plan

#### Tested with the following reproduction steps
1. Complete the onboarding flow with new account
2. After landing on the home screen navigate to ‘Recovery Phrase’
3. Navigate to this flow until you are viewing your recovery phrase.
4. Tap ‘Cancel’ and ‘Setup Later’ to exit the flow
5. Observe app crash.

### Related issues

N/A

### Backwards compatibility

Yes
